server.js

```diff
@@ -44,8 +43,6 @@ server.use(passport.initialize());
 server.use(passport.session());
 // server.use(flash());
 
-server.use('/api', router);
```

Comment:
I do not believe this should route should be deleted. Please correct me if I'm wrong but I do not see an api router import. **Are you able to test `/api` to see if it returns a response?**

Also, it looks like the `server.js` on this branch isn't up to date with master. **Are you able to try to pull from master or maybe just copy/paste the code from master into this branch's `server.js`?**

Reply:
We can resolve the merge conflicts right? That's easier than copying and pasting from diff branches

Reply:
I'm looking at masters server.js and that line's not apart of the file. Just pulled it down, and there's no merge conflicts.

Reply:
oh okay whoops didn't even see this comment. I made another comment on the pr but I probably should have done it here.. 

I am seeing it on master here: https://github.com/Lambda-School-Labs/pic_me/blob/master/server.js